frontend/javascripts/oxalis/view/action-bar/share_modal_view.tsx

```diff
@@ -84,11 +87,8 @@ export function useDatasetSharingToken(dataset: APIDataset) {
   };
 
   useEffect(() => {
-    if (!activeUser) {
-      return;
-    }
     getAndSetToken();
-  }, [dataset, activeUser]);
+  }, [dataset]);
```

Comment:
I think, technically, `activeUser` should still be part of the dependencies since `getAndSetToken` should be re-executed if it changes. At least that's my understanding of it, not sure if the linter complains about that :thinking:

Reply:
Good point! I added it now :)